ReadMe.md

```diff
@@ -90,9 +90,13 @@ English | Japanese | Kana | Notes
 --------|----------|------|------
 Development | 開発 | かいはつ |
 Developer | 開発者 | かいはつしゃ |
+Programmer | PG/エンジニア | エンジニア | 
+Engineer | PG | ぷろぐらまー |
```

Comment:
Wait it seems the japanese word for these 2 got mixed up. Read the original comment above (it was the first one). This should have been:
```
Anyways, I can separate them like so:

| Programmer | PG | プログラマー |
| Engineer | PG/エンジニア | エンジニア |

Let me know what you think of it.
```

@voltrue2 typo?